EssentialApp/EssentialAppTests/ImageCommentsUIIntegrationTests.swift

```diff
@@ -0,0 +1,204 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import Combine
+import EssentialApp
+import EssentialFeed
+import EssentialFeediOS
+
+class ImageCommentsUIIntegrationTests: XCTestCase {
+	func test_commentsView_hasTitle() {
+		let (sut, _) = makeSUT()
+
+		sut.loadViewIfNeeded()
+
+		XCTAssertEqual(sut.title, commentsTitle)
+	}
+
+	func test_loadCommentsActions_requestFeedFromLoader() {
```

Comment:
The name should be `test_loadCommentsActions_requestCommentsFromLoader`
```suggestion
	func test_loadCommentsActions_requestCommentsFromLoader() {
```

Reply:
Renamed.